### PR TITLE
User can login

### PR DESCRIPTION
**Pull request by kaylawoodbury**
# [PT Story](https://www.pivotaltracker.com/story/show/171460084)
## User Story
```
As a user
In order to access my account
I would like to be able log in
```
## Changes proposed in this pull request:
* Adds acceptance tests for user can see login form and log in
* Adds functionality to give user authentication
## What I have learned working on this feature:
*  How to authenticate a user with valid credentials
* Refreshed knowledge about stetting different states

## Screenshots
Screenshot 2020-03-05 at 10 41 45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 -update
 
 In the project directory, you can run:
-
+-- update here for semaphore test
```
Comment (oliverochman) [inline README.md]: ?

File: src/modules/authenticate.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+
+const authenticate = async (email, password) => {
+    try {
+        const response = await axios.post('auth/sign_in', {
+            email: email,
+            password: password
+        });
+        await storeAuthCredentials(response);
+        debugger
```
Comment (oliverochman) [inline src/modules/authenticate.js]: please remove debugger

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kaylawoodbury) [reply README.md]: When we were working with Faraz, he wanted me to try pushing up again to trigged semaphore to rebuild so we added some text to the readme file as a test. I can remove that if you'd like

File: src/modules/authenticate.js (hunk 2 above)
Comment (kaylawoodbury) [reply src/modules/authenticate.js]: Removed

**Review by oliverochman — APPROVED** [review]

**Pull request merged**